ui_testing/pages/analyses_page.py

```diff
@@ -10,3 +10,8 @@ def get_analysis_page(self):
         self.base_selenium.get(url=self.analyses)
         self.wait_until_page_is_loaded()
 
+    def filter_by_analysis_number(self, filter_text):
+        self.info('Filter by analysis number : {}'.format(filter_text))
+        self.open_filter_menu()
+        self.filter_by(filter_element='analysis_page:analysis_no_filter', filter_text=filter_text, field_type='text')
+        self.filter_apply()
```

Comment:
this method already in the orders_page so you call it from the orders_page why you add it one more time to the analyses_page

Reply:
as filter elements in analysis page differs from orders' page.